app/views/api/v1/release_packages/pypi/simple/index.html.haml

```diff
@@ -0,0 +1,10 @@
+-# frozen_string_literal: true
+- title = "Packages for #{account.name}"
+- content_for(:title) { title }
+
+%h1= title
+
+- packages.each do |package|
+  - url = url_for([:v1, account, :release_packages, :pypi_simple_package, id: package.id, format: nil, trailing_slash: true])
+
+  = link_to(package.code, url, data: package.metadata)
```

Comment:
Should we require that all products have a code, or write a script to set a default?

```ruby
Product.find_each do |product|
  product.update!(code: product.name.parameterize)
end
```

Reply:
A simple way to do this would be to insert a `package.read` permission for every role and token that has a `artifact.read` permission.

Reply:
I feel like we should do both. PEP 503 requirs that this match the package name.

Reply:
We should replace a product's `code` with a `code` column on the new package active record.